### DISCUSSION TITLE
10.05.2023 Area A Data Modeling

### DISCUSSION THREAD

**Discussion by hampusnasstrom** [FAIRmat-NFDI/AreaA-data_modeling_and_schemas#47] [Data Modeling Meeting]
## Meeting notes
- Discussion points for 16.5
  - Whole experiment
    - Plan, execute, and result
    - Synthesis, measurement, and calculation
  - Workflow
  - Single vs multiple archives
    - Normalizer for grouping
    - Micha's work
  - Separate ELN (invite Markus for an hour)
    - Workflow (Entity-Activity)
    - "System" (Entity-Entity)
    - Reprocessing an ELN
      - Where is ELN data stored
  - Base classes
    - Not all ELN
    - Inheritance vs composition
  - Sample
    - Propagating properties
    - System (layers, phases, etc.)
    - Workflow of multiple samples and activities
- Markus (Area D) points:
  - Where we put our base classes
    - ELN, plugin
  - Separate ELN
  - Workflow
    - Workflow (Entity-Activity)
    - "System" (Entity-Entity)
  - JMESpath

### Agenda for 16.5
1. Base classes
2. Experiment
3. Sample
4. Workflow
5. Area D tasks

**Comment by Pepe-Marquez** [comment]
- Area AGeneral Meeting
	- Calculation. Do we need a base class for this? If so, how to align it with an ML. How to make them searchable?
	- Micha Goette
		- adding links to ontological information
		- links:
	- `nomad.datamodel.metainfo`
	- BaseSections
		- Sections can have categories
		- propose to go in `base_section.py` in `nomad.datamodel.metainfo`:
			- BaseSection
				- Entity
					- System
						- Substance
						- Ensemble (name?)
					- Instrument
				- Activity
					- Process
					- Measurement
					- Calculation/Analysis/MLmodels?
			- ArchiveSection
				- PublicationReference
				- StructureFile
	- Results Questionable...
		- method `results.eln.method` ask Luca for consolidation
			- do we need to ask for categorizing this?
		- id (human identifier?) replacement for `results.eln.lab_id`
		- remove `eln.instrument`

**Comment by andreaa93** [comment]
TODOs:

- where to place our Synthesis plugin? (either in Nomad source code or as pip installable dependency)
- refactoring of results section (erase eln subsection)
- split substance and substance with CAS
- ENSEMBLE or SAMPLE (discuss with other areas again)
